src/Mono.Android/Mono.Android.targets

```diff
@@ -260,4 +268,26 @@
       <FileWrites Include="$(IntermediateOutputPath)CheckApiCompatibility.stamp" />
     </ItemGroup>
   </Target>
+
+  <Target Name="UpdateExternalDocumentation">
+    <Exec Command="git clean -xdf obj" WorkingDirectory="$(MSBuildThisFileDirectory)" />
+    <Exec
+        Command="time msbuild /restore /p:IncludeAndroidJavadoc=True /bl"
```

Comment:
`time` is not available on Windows. Would it make sense to change it to use MSBuild task or at least not use `time` conditionally on Windows? (same also few lines down)

Reply:
`time` has been removed.